admin-dev/themes/new-theme/scss/pages/_product_page.scss

```diff
@@ -534,3 +530,86 @@
     color: $gray-medium;
   }
 }
+
+#attributes-list {
+  .attribute-group {
+    border: rem(1px) solid $gray-light;
+    border-bottom-width: 0;
+
+    &:last-child {
+      border-bottom-width: rem(1px);
+    }
+
+    .attribute-group-name {
+      display: block;
+      font-weight: $font-weight-semibold;
+      color: $gray-dark;
+      padding: rem(7px);
+
+      &:hover {
+        text-decoration: none;
+      }
+
+      &:after {
+        content: "\E316";
+        font-family: "Material Icons";
+        float: right;
+      }
+
+      &.collapsed:after {
+        content: "\E313";
+      }
+    }
+
+    .attributes {
+      border-top: 2px solid $gray-light;
+      padding: em(7px);
+      height: rem(158px);
+      overflow: auto;
+
+      .attribute {
+        .js-attribute-checkbox {
+          display: none;
+
+          +.attribute-label {
```

Comment:
It's to select the first `.attribute-label` element right next to `.js-attribute-checkbox`